node-optional.el9.repo

```diff
@@ -2,16 +2,31 @@
 
 [onn-baseos]
 name = oVirt Node Optional packages from CentOS Stream $releasever - BaseOS
-baseurl = http://mirror.stream.centos.org/9-stream/BaseOS/$basearch/os/
+metalink = https://mirrors.centos.org/metalink?repo=centos-baseos-$stream&arch=$basearch&protocol=https,http
```

Comment:
Perhaps add something about this to the commit message, if you do not want to split to its own patch

Reply:
I'll move to separate patch